.circleci/requirements/dev-requirements-py37.txt

```diff
@@ -1,5 +1,5 @@
-dash_core_components>=0.40.2
-dash_html_components==0.12.0rc3
+dash_core_components>=0.43.0
+dash_html_components==0.13.4
 dash-flow-example==0.0.3
 dash-dangerously-set-inner-html
 git+git://github.com/plotly/dash-renderer@master#egg=dash_renderer
```

Comment:
I think renderer can go back to the published version now... this was supposed to be temporary during the no-events push (sorry about that!)